src/server/utils/validations.js

```diff
@@ -0,0 +1,68 @@
+'use strict'; // eslint-disable-line
+const moment = require('moment');
+const _ = require('lodash');
+
+const validations = {
+  /**
+   * Validate a string as and convert to a valid format.
+   * @param {string} inputDob The date of birth string to validate / convert.
+   * @returns {string} If input was valid a valid SSN if not valid an empty string.
+   */
+  dateOfBirth: (inputDob) => {
+    let convertedDob = '';
+    if (_.isEmpty(inputDob) || !_.isString(inputDob)) {
+      return convertedDob;
+    }
+    const parsedDob = new Date(inputDob);
+    if (moment().isBefore(parsedDob)) {
+      return convertedDob;
+    }
+    convertedDob = moment(parsedDob).format('MM/DD/YYYY');
+    return convertedDob;
+  },
+  /**
```

Comment:
I was sorta expect function statement. For example:

```
function validateString() {
}

module.exports = { validateString, dateOfBirth, ...};
```

It's more verbose but feels a bit more explicit.  Push back if it annoys you. This kind of style is purely preference and I don't want to dictate...